plugin/storage/factory_config.go

```diff
@@ -56,13 +57,19 @@ func FactoryConfigFromEnvAndCLI(args []string, log io.Writer) FactoryConfig {
 	if spanStorageType == "" {
 		spanStorageType = cassandraStorageType
 	}
-	depStoreType := os.Getenv(DependencyStorageTypeEnvVar)
-	if depStoreType == "" {
-		depStoreType = spanStorageType
+	spanWriterTypes := strings.Split(spanStorageType, ",")
+	if len(spanWriterTypes) > 1 {
+		fmt.Fprintf(log, "WARNING: Since multiple storage types have been listed,"+
```

Comment:
It seems like this warning could be easily missed. 
Is it difficult to require the user to explicitly specify a read and archive storage type when multiple storage types are specified?